app/src/main/java/co/techmagic/hr/presentation/time_tracker/info/TimeInfoContract.kt

```diff
@@ -0,0 +1,25 @@
+package co.techmagic.hr.presentation.time_tracker.info
+
+import com.techmagic.viper.Presenter
+import com.techmagic.viper.Router
+import com.techmagic.viper.View
+
+interface TimeInfoView : View {
+    fun showLoading()
+    fun hideLoading()
+    fun showReports(reports: List<TimeReportViewModel>)
+}
+
+interface TimeInfoPresenter : Presenter {
+
+}
+
+interface TimeInfoRouter : Router {
+
+}
+
+data class TimeReportViewModel(
```

Comment:
We already have class `UserReportViewModel.kt`. I think it will confuse people, if they see this code first time.
I suggest to rename this class, for example, to `WorkingTimeInfoViewModel` or something else.